README.md

```diff
@@ -303,3 +303,57 @@ The `./sdk-generation` directory contains code and config for our SDK generation
 | php | PHP >= 7.2
 | phpold | Does not have XML support |
 | node | Highly recommended to generate with a specified spec |
+
+
+## Api Doc preprocessor
+
+The `/preprocessor` directory contains `apiDocPreProcessor.jar` file that can remove marked `JSON`
+and/or `XML`objects in api doc spec payload. This was developed to have  a possibility to create 
+`internal` and `external` API doc.
+To mark `JSON` object to be removed you need to add `internal:true` 
+field to object that should be removed:
+
+1) Example for JSON Object:
+```
+ {...},
+ "AccountId": {
+    "type": "integer",
+    "format": "int32",
+    "internal": true, <-- marker to remove object
+ },
+ {...}
+```
+Added `internal` property that is set as true will remove `AccountId` object from payload.
+
+2) Example for XML Object:
+```
+    ...
+    <Contact internal=`true`> <-- contains marker
+      <FirstName>Sanjay</FirstName>
+      <LastName>Rao</LastName>
+      <Phone>9195441234</Phone>
+      <Email>srao@bandwidth.com</Email>
+    </Contact>
+    <AltSpid>X455</AltSpid>
+    <SPID>9999</SPID>
+    <PortCarrierType internal=`true`>WIRELINE</PortCarrierType> <--contains marker
```

Comment:
AFAIK openapi doesnt support XML with both elements and attributes 

Does this processor read/write the xml values from/to the example field? Or the model schema?